.testcontainer/ops.json

```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "uuid": "0a9fa342-3139-49d7-8acb-fcf4d9c1f0ef",
+    "name": "DanTheTechMan",
```

Comment:
A comment could be added to this file or an accompanying README to instruct developers on modifying the op list

Reply:
Addressed